### PR TITLE
docs: update repo name across docs

### PR DESCRIPTION
**Pull request by salaxander**
## Description
May have missed some, but it does look like there are some that need to stay as is

### CODE REVIEW

File: site/src/content/docs/ref/github-action.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar:
 tableOfContents: false
 ---
 
-The [setup-zarf](https://github.com/defenseunicorns/setup-zarf) Github action is an officially supported action to install any version of Zarf and it's `init` package with zero added dependencies.
+The [setup-zarf](https://github.com/zarf-dev/setup-zarf) Github action is an officially supported action to install any version of Zarf and it's `init` package with zero added dependencies.
```
Comment (AustinAbro321) [inline site/src/content/docs/ref/github-action.mdx:8]: This action hasn't been moved yet

File: site/src/content/docs/ref/github-action.mdx [hunk 2]
```diff
@@ -21,12 +21,12 @@ jobs:
           fetch-depth: 1
 
       - name: Install Zarf
-        uses: defenseunicorns/setup-zarf@main # use action's main branch
+        uses: zarf-dev/setup-zarf@main # use action's main branch
```
Comment (AustinAbro321) [inline site/src/content/docs/ref/github-action.mdx:24]: ditto

File: site/src/content/docs/ref/github-action.mdx [hunk 3]
```diff
@@ -21,12 +21,12 @@ jobs:
...
         with:
           version: v0.32.2 # any valid zarf version, leave blank to use latest
 
       - name: Create the package
         run: zarf package create --confirm
 ```
 
-More examples are located in the action's [README.md](https://github.com/defenseunicorns/setup-zarf#readme)
+More examples are located in the action's [README.md](https://github.com/zarf-dev/setup-zarf#readme)
```
Comment (AustinAbro321) [inline site/src/content/docs/ref/github-action.mdx:32]: ditto

**Review by AustinAbro321 — CHANGES_REQUESTED** [review]

File: site/src/content/docs/faq.mdx [hunk 4]
```diff
@@ -4,29 +4,29 @@ title: FAQ
 
 ## Who is behind this project?
 
-Zarf was built by the developers at [Defense Unicorns](https://www.defenseunicorns.com/) and an amazing community of contributors.
+Zarf was built by the developers at [Defense Unicorns](https://www.zarf-dev.com/) and an amazing community of contributors.
```
[the remark this answers is not in the document]
Comment (phillebaba) [reply site/src/content/docs/faq.mdx:7]: zarf-dev.com also needs to function properly first.

File: site/src/content/docs/commands/zarf_package_pull.md [hunk 5]
```diff
@@ -19,13 +19,13 @@ zarf package pull PACKAGE_SOURCE [flags]
 ```
 
 # Pull a package matching the current architecture
-$ zarf package pull oci://ghcr.io/defenseunicorns/packages/dos-games:1.0.0
```
Comment (phillebaba) [inline site/src/content/docs/commands/zarf_package_pull.md:22]: This has not been moved.

File: site/src/content/docs/faq.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (salaxander) [reply site/src/content/docs/faq.mdx:7]: Oh my bad, this should be discarded

File: site/src/content/docs/ref/deploy.mdx [hunk 6]
```diff
@@ -117,7 +117,7 @@ $ zarf connect [service name]
 
 :::note
 
-You can also specify a package locally, or via oci such as `zarf package deploy oci://defenseunicorns/dos-games:1.0.0-$(uname -m) --key=https://zarf.dev/cosign.pub`
+You can also specify a package locally, or via oci such as `zarf package deploy oci://zarf-dev/dos-games:1.0.0-$(uname -m) --key=https://zarf.dev/cosign.pub`
```
[the remark this answers is not in the document]
Comment (AustinAbro321) [reply site/src/content/docs/ref/deploy.mdx:120]: This should stay the same, but this is the package not the image so it's actually hosted on the defense unicorn github

[the remark this answers is not in the document]
Comment (AustinAbro321) [reply site/src/content/docs/ref/deploy.mdx:120]:
```suggestion
You can also specify a package locally, or via oci such as `zarf package deploy oci://defenseunicorns/dos-games:1.0.0 --key=https://zarf.dev/cosign.pub`
```
though we should delete the uname as we no longer append arch to our oci packages

File: site/src/content/docs/tutorials/3-deploy-a-retro-arcade.mdx [hunk 7]
```diff
@@ -22,7 +22,7 @@ Before beginning this tutorial you will need the following:
 
 ## Deploying the Arcade
 
-1. The `dos-games` package is easily deployable via `oci://` by running `zarf package deploy oci://defenseunicorns/dos-games:1.0.0-$(uname -m) --key=https://zarf.dev/cosign.pub`.
+1. The `dos-games` package is easily deployable via `oci://` by running `zarf package deploy oci://zarf-dev/dos-games:1.0.0-$(uname -m) --key=https://zarf.dev/cosign.pub`.
```
Comment (AustinAbro321) [inline site/src/content/docs/tutorials/3-deploy-a-retro-arcade.mdx:25]:
```suggestion
1. The `dos-games` package is easily deployable via `oci://` by running `zarf package deploy oci://defenseunicorns/dos-games:1.0.0 --key=https://zarf.dev/cosign.pub`.
```

**Review by schristoff — APPROVED** [review]

**Review by phillebaba — APPROVED** [review]

**Pull request merged**